src/tiledimage.js

```diff
@@ -419,6 +419,15 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         return new $.Point(this.source.dimensions.x, this.source.dimensions.y);
     },
 
+    /**
+     * @returns {OpenSeadragon.Point} The TiledImage's content size, in window coordinates.
+     */
+     getSizeInWindowCoordinates: function() {
+        var topLeft = this.viewport.imageToWindowCoordinates(new $.Point(0, 0));
```

Comment:
Using the `viewport` version will only work properly if there's only a single image in the viewer. Instead use the TiledImage's version (i.e. just omit `viewport.`).

Reply:
Oh cool, thanks for pointing this out!

Reply:
Cool, thanks for pointing this out :D